### PR TITLE
Fixed Get Open Sales Orders with Item Filter

### PR DESCRIPTION
**Pull request by javierwonghf**
Previously Get Open Sales Orders with Item Filter does not filter properly and instead returns Sales Orders with all Items.

### CODE REVIEW

File: erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def get_open_sales_orders(self):
 				and so.company = %(company)s
 				and so_item.qty > so_item.delivered_qty {0}
 				and (exists (select name from `tabItem` item where item.name=so_item.item_code
-					and (item.is_pro_applicable = 1 or item.is_sub_contracted_item = 1 {1}))
+					and ((item.is_pro_applicable = 1 or item.is_sub_contracted_item) = 1 {1}))
```
Comment (anandpdoshi) [inline erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.py]: Shouldn't the closing bracket bet after `= 1`?

Comment (anandpdoshi) [inline erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.py]: I will fix and merge it :+1:

**Comment by anandpdoshi** [comment]
cherry-picked commit and merged manually: https://github.com/frappe/erpnext/commit/3d1f01ffc1aaa26cddfc5ad805178e744c9fe24c

**Comment by bohlian** [comment]
haha yes! sorry and thanks @anandpdoshi

**Pull request closed without merging**